GameServer/Logic/Game.cs

```diff
@@ -73,13 +62,34 @@ public bool Initialize(Address address, string baseKey)
             _server = new Host();
             _server.Create(address, 32, 32, 0, 0);
 
-            var key = System.Convert.FromBase64String(baseKey);
+            var key = Convert.FromBase64String(baseKey);
             if (key.Length <= 0)
+            {
                 return false;
+            }
 
             Blowfish = new BlowFish(key);
             PacketHandlerManager = new PacketHandlerManager(_logger, Blowfish, _server, _playerManager);
-            Map = new SummonersRift(this);
+
+            switch (Config.GameConfig.Map)
```

Comment:
What we really want to do is have a string identifier for each map, that would indicate the content pack it's in + the map name.

So instead of using numbers, we'd want to use something like `LeagueSandbox-Default-Map10`
This would indicate the map is from the author `LeagueSandbox`, in a content pack called `Default`, and the map is called `Map10`

These could also be called something more sensible like `LeagueSandbox-Default-TwistedTreeline`, but that doesn't really matter. The way the strings are built will come from the content folder names, from [here](https://github.com/LeagueSandbox/LeagueSandbox-Default/tree/master/Maps)

You don't need to implement the entire system yet, but what you'll want to do for now essentially is:
```CS
var mapRegister = new Dictionary<string, Map>
{
    {"LeagueSandbox-Default-Map1": typeof(SummonersRift)},
    // {"LeagueSandbox-Default-Map8": typeof(CrystalScar)},
    {"LeagueSandbox-Default-Map10": typeof(TwistedTreeline)},
    // {"LeagueSandbox-Default-Map11": typeof(ReworkedRift)},
    {"LeagueSandbox-Default-Map12": typeof(HowlingAbyss)},
}
Map = (Map)Activator.CreateInstance(mapRegister[Config.GameConfig.Map], this);
```

That method should be extracted to it's own method as well IMO

All maps should use the same class in the end once everything works as intended, but this would be the best solution for now. You'll also want to change the default config template of course.